src/main/java/com/ioet/bpm/bpmorganizationsapi/boundaries/OrganizationController.java

```diff
@@ -8,20 +8,22 @@
 import io.swagger.annotations.ApiResponse;
```

Comment:
The package name is really weird. I suggest com.ioet.bpm.organizations.boundaries instead.

Reply:
I will change that.